sonic-xcvrd/xcvrd/xcvrd_utilities/media_settings_parser.py

```diff
@@ -208,9 +208,13 @@ def get_media_settings_for_speed(settings_dict, lane_speed_key):
     return settings_dict.get(LANE_SPEED_DEFAULT_KEY, {})
 
 
-def get_media_settings_value(physical_port, key):
+def get_media_settings_value(physical_port, key, gearbox_side=None):
```

Comment:
@j-mazurek please rename this to `get_asic_media_settings()` and keep it separate from get_gb_media_settings()` (see my comments below)